addons/mail/static/src/models/list_field_activity_view.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'ListFieldActivityView',
+    fields: {
+        id: attr({
+            identifying: true,
+        }),
+        summaryText: attr({
+            compute() {
+                if (this.thread.activities.length === 0) {
+                    return '';
+                }
+                if (this.webRecord.data.activity_exception_decoration) {
+                    return this.env._t("Warning");
+                }
+                if (this.webRecord.data.activity_summary) {
+                    return this.webRecord.data.activity_summary;
+                }
+                if (this.webRecord.data.activity_type_id) {
+                    return this.webRecord.data.activity_type_id[1]; // 1 = display_name
+                }
+                return '';
+            },
```

Comment:
Could `default: ''` so it's just `clear()`

Reply:
changed but I'm not a big fan